testing/test_pathlib.py

```diff
@@ -89,3 +89,15 @@ def renamed_failed(*args):
     lock_path = get_lock_path(path)
     maybe_delete_a_numbered_dir(path)
     assert not lock_path.is_file()
+
+def test_long_path_during_cleanup(tmp_path):
+    """Ensure that deleting long path will be able to be deleted"""
+    path = tmp_path / ("a" * 200)
+    if sys.platform == "win32":
+        dirname = path.resolve()
+        dirname = "\\\\?\\" + str(path)
```

Comment:
A raw string would be nice here, like you have in the diff above.

Reply:
I think I may have been confusing, I meant it'd be nicer to switch from `"\\\\?\\"` to `r"\\?\"` but you went the other way :smile: 

It's not very important though.

Reply:
I forgot that you can't include a trailing blackslash in a string literal in python.